webprojects/fundraising.md

```diff
@@ -0,0 +1,27 @@
+# Fundraising website: how to localize and test
+
+[donate.mozilla.org](https://donate.mozilla.org) is our platform to collect donations. The main form is for Mozilla Foundation, but it also hosts a separate form for donations to Thunderbird.
+
+This website relies on third parties to handle payment. The localization of the payment form for debit/credit card payment is managed by Stripe and we do not have control.
+
+## Testing
+
+Submitted translations are automatically deployed to the staging server, so you can test them a few minutes after they have been translated.
+
+* Staging server: https://donate.mofostaging.net/​.
+* Production server: https://donate.mozilla.org/.
+
+### Testing the payment flow
+
+You may want to test the complete user journey on the website, but you don’t have to make an actual donation for that. Stripe has a [test mode](https://stripe.com/docs/testing#cards) that is enabled on the staging server only. Simply use the payment info from this page on the staging server and you should be able to simulate a payment.
```

Comment:
Can you rephrase the second sentence so it's clear what happens, and what localizers are supposed to test? Not sure I understand, not having seen the website.